catkit2/services/nkt_superk/nkt_superk.py

```diff
@@ -46,6 +46,36 @@ class Evo(Enum):
     REG_MAC_ADDRESS = 0xB3
 
 
+class Fianium(Enum):
+    DEVICE_ID = 15
+
+    #REG_BASE_TEMPERATURE = 0x17
+    #REG_SUPPLY_VOLTAGE = 0x1D
+    #REG_EXTERNAL_CONTROL_INPUT = 0x94
```

Comment:
If you have double-checked that the Fianium doesn't have any of these, delete them all. Together with the "added" comments, since that is obvious from the code.